deepspeed/launcher/runner.py

```diff
@@ -376,7 +376,7 @@ def parse_inclusion_exclusion(resource_pool, inclusion, exclusion):
     node_configs = parse_node_config_list(inclusion)
 
     for hostname, slots in resource_pool.items():
-        active_resources[hostname] = node_configs[hostname] if hostname in node_configs else list(range(slots))
+        active_resources[hostname] = node_configs[hostname] if hostname in node_configs and node_config[hostname]!=[]  else list(range(slots))
```

Comment:
@stephen-nju - can you run our pre-commit formatter?  It will require spaces around the `!=`

Reply:
sorry , I don't know how to use this tools

Reply:
I've pushed fixes for this.  The steps to run the pre-commit formatter are [here](https://github.com/microsoft/DeepSpeed/blob/master/CONTRIBUTING.md#prerequisites)

Reply:
thanks